src/libraries/System.Drawing.Common/tests/Printing/PrinterSettingsTests.cs

```diff
@@ -437,6 +437,7 @@ public void SupportsColor_ReturnsExpected()
         {
             var printerSettings = new PrinterSettings();
             bool supportsColor = printerSettings.SupportsColor;
+            Assert.True(supportsColor);
```

Comment:
nit: you can assert directly on printerSettings.SupportsColor

Reply:
Oh right this was for testing locally forgot to cleanup.

Reply:
I'm not sure I understand how this works. You are testing a random printer, how can you assert anything about it?

Reply:
I tests the default printer if available which in a fresh install of Windows (which is how the CI machines are setup) is the PDF printer. I can remove it and leave the test as it was, I was also hesitant of adding this assert as the Printer can be different from machine to machine. 

I could force it to get the PDF printer instead as well.

Reply:
I would prefer something like this:
                var printerSettings = new PrinterSettings() { PrinterName = "Microsoft Print to PDF" };
                Assert.True(printerSettings.SupportsColor);

Reply:
Yeah that is what I was thinking with: `I could force it to get the PDF printer instead as well.`